src/elements/content-sidebar/activity-feed/task-new/Task.scss

```diff
@@ -39,6 +39,12 @@ $bcs-task-left-offset: 10 * $bcs-task-grid + 2px;
 
 .bcs-Task-statusContainer {
```

Comment:
Ideally, we should be avoiding class names with the word "container" in them. The root class, `bcs-Task-status`, should be the container.